src/analysis/output.js

```diff
@@ -33,6 +33,10 @@ export const COMPAT_INFO = {
 export type OutputNode = {|
   +address: $ReadOnlyArray<string>,
   +cred: number,
+  // Full cred over time (aligned with output interval boundaries).
+  // It's optional because it inflates the output size a lot -- we'll
+  // want to filter out low-cred nodes for large projects
+  +credOverTime: ?$ReadOnlyArray<number>,
```

Comment:
Nit: this isn't great at self-describing.
By that I mean, it will be tough to figure out from the output which filters were used. Some kind of filter details (a mode enum? set of whitelisted prefixes?) would be good to include. Since no filtering is actually implemented in this version, we could also make it required till we do support it.

Reply:
Ack. My intention is to let the filter be arbitrary logic, so there isn't a clean way to encode the result. I suppose we could standardize around a mode enum, but that adds a lot of complexity for the consumer to keep track of all the potential filter modes. 

As an example of a reasonably complex filter, we might have the composition of the following rules:
- keep every user node
- keep every node with greater than > 30 total cred
- keep all pull requests, issues, and topics from the last month

Would be a pain to track that via mode enums.

Reply:
Which makes sense but is a trade-off, because by necessity will require an out-of-band description of how this output was produced, should you want to have reproducibility of it. And in many cases you do want reproducibility, to support any claims you're deriving from it, or to expose it in a canonical instance.